README.md

```diff
@@ -12,13 +12,26 @@ A **platform** is software that orchestrates a lifecycle to make buildpack funct
 
 ## Notational Conventions
 
+### Key Words
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
 
 The key words "unspecified", "undefined", and "implementation-defined" are to be interpreted as described in the [rationale for the C99 standard](http://www.open-std.org/jtc1/sc22/wg14/www/C99RationaleV5.10.pdf#page=18).
 
 An implementation is not compliant if it fails to satisfy one or more of the MUST, MUST NOT, REQUIRED, SHALL, or SHALL NOT requirements for the protocols it implements.
 An implementation is compliant if it satisfies all the MUST, MUST NOT, REQUIRED, SHALL, and SHALL NOT requirements for the protocols it implements.
 
+### Operating System Conventions
+
+When a word or bullet point is prefixed with a <a name="linux-only">†</a>, it SHALL be assumed to apply only to Linux stacks.
+
+When a word or bullet point is prefixed with a <a name="windows-only">‡</a>, it SHALL be assumed to apply only to Windows stacks.
```

Comment:
is this a common notation?

Reply:
At the time introduced this, @ekcasey and I couldn't find any existing notation patterns for this type of consistent dichotomy so we used these symbols as start. If you have any other suggestions or prior art, I'd be happy to consider changing it.

Reply:
👍 no i don't have a better suggestion.

Reply:
Unfortunately, the OCI specs I'm my opinion don't go very far in documenting all the Windows differences. There are a couple notes:

* [runtime-spec/config.md](https://github.com/opencontainers/runtime-spec/blob/ad53dcdc39f1f7f7472b10aa0a45648fe4865496/config.md)
  * Inconsitent conventions "On Windows, ...", "Windows:", "Example (Windows)" 

* [runtime-spec/config-windows.md](https://github.com/opencontainers/runtime-spec/blob/ad53dcdc39f1f7f7472b10aa0a45648fe4865496/config-windows.md#windows-specific-container-configuration)
  * Stand-alone section just for Windows-specific stuff

* [image-spec/layer.md](https://github.com/opencontainers/image-spec/blob/9f4348abedbe4415e6db1f08689fa7588045d982/layer.md#platform-specific-attributes)
  * Windows-specific paragraph platform specific references

* [image-spec/image-index.md](https://github.com/opencontainers/image-spec/blob/243ea084a44451d27322fed02b682d99e2af3ba9/image-index.md#image-index-property-descriptions)
  * Some windows mentions manifest/platforms/os.features

In all these examples, I personally find it hard to parse them to determine what exactly needs and doesn't need to be implemented for Windows and feel like a consistent global/local notation is preferable. Happy to hear other thoughts though